### PR TITLE
add en-help

### PR DESCRIPTION
**Pull request by gebilxs**
Change official website, update description to English

### CODE REVIEW

**Comment by zhangguanzhang** [comment]
可以在之前的pr上rebase -i HEAD~2 后 把第二行的pick改成s，完成rebase后git push -f就行了，不用新开一个pr

**Comment by gebilxs** [comment]
🔭学到了谢谢😄

File: cmd/sealos/cmd/run.go [hunk 1]
```diff
@@ -36,10 +36,15 @@ var contact = `
     \::/  /       \:\__\         /:/  /       \:\__\    \::/  /       \::/  /   
      \/__/         \/__/         \/__/         \/__/     \/__/         \/__/  
 
-                  官方文档：www.sealyun.com
+                  Website :https://www.sealos.io/
+                  Address :github.com/labring/sealos
+                  Problem :github.com/labring/sealos/issues
+                  QQ Group:	 98488045
+
+                  官方文档：https://www.sealyun.com/
```
Comment (fanux) [inline cmd/sealos/cmd/run.go:44]: Just delete Chinese items.

File: cmd/sealos/cmd/run.go [hunk 2]
```diff
@@ -36,10 +36,15 @@ var contact = `
     \::/  /       \:\__\         /:/  /       \:\__\    \::/  /       \::/  /   
      \/__/         \/__/         \/__/         \/__/     \/__/         \/__/  
 
-                  官方文档：www.sealyun.com
+                  Website :https://www.sealos.io/
+                  Address :github.com/labring/sealos
+                  Problem :github.com/labring/sealos/issues
```
Comment (fanux) [inline cmd/sealos/cmd/run.go:41]: Delete this item and QQ Group.

**Review by cuisongliu — APPROVED** [review]

**Pull request merged**